tests/test_conflicting_formatters.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from collections.abc import Generator
+from pathlib import Path
+
+import pytest
+from _pytest.fixtures import SubRequest
+
+from pydocstringformatter import _formatting
+from pydocstringformatter import _testutils as test_utils
+from pydocstringformatter._utils import UnstableResultError
+from pydocstringformatter.run import _Run
+
+
+@pytest.fixture(scope="function")
+def patched_run(request: SubRequest) -> Generator[type[_Run], None, None]:
```

Comment:
`request` is now unused right?

Reply:
still used in L22 to use it as a reusable fixture

Reply:
Oh my bad, but can't that be a normal param? Something like:
```suggestion
def patched_run(formatters: list[str]) -> Generator[type[_Run], None, None]:
```

Reply:
well .... if we moved it from being a fixture to a context manager ... and then just used it directly in the test

Reply:
I think even normal fixtures can return `Generators`? We use them in `pylint` as well. And we pass normal parameters then as well.

Reply:
Ok, reading the pytest documentation made clear how to ... but it is awful to read ...
Changed it to a simple contextlib manager, let me know what you think (same result, much more readable)